File: java-main/java-main/Array/Ex3/p7.java
//WAP to find a Strong number from an array and return its index.

import java.io.*;

class Demo{

	public static void main(String[] args)throws IOException{

		BufferedReader br= new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter array size");

		int size=Integer.parseInt(br.readLine());

		int arr[]=new int[size];

		System.out.println("Enter array elements");

		for(int i=0;i<size;i++){
			arr[i]=Integer.parseInt(br.readLine());
		}

		for(int i=0;i<size;i++){
			int sum=strong(arr[i]);

			if(arr[i]==sum){
				System.out.println("Strong " + arr[i] + " found at index: " + i);
			}
		}
	}

	static int strong(int x){

		int temp=x;
		int sum=0;

		while(temp!=0){
			int fact=1;

			int rem=temp%10;
			for(int i=1;i<=rem;i++){
				fact=fact*i;
			}
			sum=sum+fact;

			temp=temp/10;
		}
		return sum;
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/practical2/p7.c
#include<stdio.h>
void main(){

	int num;
	int i;
	int rem=0,sum=0;

	printf("Enter the number\n");
	scanf("%d",&num);

	while(i<=num){
		rem=num%10;
		num=num/10;
		sum=sum+rem;
		i+2;
	}
	printf("sum is :%d\n",sum);
}

